src/utils/sourcemaps.rs

```diff
@@ -327,7 +327,15 @@ impl SourceMapProcessor {
             };
 
             let sourcemap_url = match discover_sourcemaps_location(contents) {
```

Comment:
should we move this to this function?

I’m a bit unsure, as in symbolicator when we scrape the `sourceMappingURL` from the file because we had no bundle "header" to take it from, we have to strip these when looking up the sourcemap in the bundles we have, but having these present in case we fall back to scraping might actually be beneficial.

When in doubt, I would rather just strip them reliably as early as possible.